src/test/kotlin/domain/lotto/game/LottoGameTest.kt

```diff
@@ -19,6 +21,14 @@ class LottoGameTest {
         lottoGame = LottoGame(LottoMachine())
     }
 
+    @Test
+    fun `manualLottoList와 금액이 주어졌을 때, purchasedLottos() 호출시, 구매금액 만큼의 PurchasedLotto를 반환한다`() {
```

Comment:
```suggestion
    fun `5천원으로 5장을 구매한다`() {
```
테스트 코드에는 가급적이면 내부 정보를 드러내지 않고, 어떤 것을 테스트하고자 하는지를 나타내는 것이 좋습니다.
만약 몇개월이 지난 후 purchasedLottos 함수명을 변경하게 된다면, 이 함수명도 변경할 수 있을까요? (이하 관련 내용 동일)